python/cuspatial/cuspatial/core/binpreds/feature_crosses.py

```diff
@@ -35,7 +45,7 @@ def _preprocess(self, lhs, rhs):
     (MultiPoint, MultiPoint): NotImplementedPredicate,
     (MultiPoint, LineString): NotImplementedPredicate,
     (MultiPoint, Polygon): NotImplementedPredicate,
-    (LineString, Point): NotImplementedPredicate,
+    (LineString, Point): ImpossiblePredicate,
```

Comment:
Because GeoPandas returns False for the combinations of LineString, Point:
![lines-dont-cross-points](https://user-images.githubusercontent.com/410931/230460474-4c5c0dc4-5d47-4d8c-9920-857e579ce027.png)